nana/modules/stickers.py

```diff
@@ -29,15 +28,15 @@
 """
 
 
-@app.on_message(Filters.user(AdminSettings) & Filters.command("kang", Command))
+@app.on_message(filters.user(AdminSettings) & filters.command("kang", Command))
```

Comment:
Codacy found an issue: [kang_stickers is too complex (20) (MC0001)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330175)